inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -143,6 +150,25 @@ def arxiv_refextract(obj, eng):
         obj.log.error("Not able to download and process the PDF")
 
 
+def arxiv_derive_inspire_categories(obj, eng):
+    """Derive inspire-categories from arxiv-categories
+
+    :param obj: Workflow Object to process
+    :param eng: Workflow Engine processing the object
+    """
+    obj.data.setdefault('inspire_categories', [])
+
+    for arxiv_category in get_value(obj.data['arxiv_eprints'][0], 'categories'):
+        inspire_category = classify_field(arxiv_category)
+        if inspire_category:
+            category = {
+                'source': 'arxiv',
+                'term': inspire_category,
+            }
+            if category not in obj.data['inspire_categories']:
```

Comment:
for now it's not needed, but in the future, we will handle updates where inspire categories will already have been assigned. So probably safer to leave it.